blocks/table/table.css

```diff
@@ -131,3 +145,63 @@ main .table table tbody tr:nth-child(even) {
     visibility: visible;
   }
 }
+
+main .table.weather table {
+  padding: 30px;
+  border: none;
+  font-size: 2vw;
```

Comment:
using `vw` as a font unit makes this table nearly impossible to see on mobile sized screens. if the 2vw is _really_ desired, i would consider wrapping it in a `max()`. something like `max(var(--body-font-size-xs), 2vw)`, maybe?

Reply:
I noticed we're not leveraging vw units in general.
Do we have another approach to mobile views?

In testing 2vw appeared legible on mobile devices. 
Once width exceeds 1100 I switched this to one of the variable font sizes.

Would you suggest wrapping all such relative units in a max?

Reply:
personally, I have _never_ seen `vw` as a font unit in a production site. i don't know if i have a good answer for "another approach." the site was migrated using the existing font size specs (pixels), and the font size variables change depending on viewport width. i definitely don't disagree that having more "responsive" font is a good idea, while ensuring that the text is still readable.

i guess maybe legible is subjective. for text thats more than a 2-3 word caption, i wouldn't have text smaller than 9-12 pt/`12-16px`. on a samsung galaxy, `2vw` is still less than `9px`. my pixel is less than `8px`. 

honestly, no, i wouldn't recommend that, but mostly because i don't recommend using `vw` as a font unit. i think `rem` is probably a better way to set relative font size.

Reply:
Thanks Katie, the issue comes from needing to see all table content legibly within any viewport.

I had tried to use rem units but I noticed we don't set font-size on the root element.

I tried em units but on the body it seems we are always setting font-size to: var(--body-font-size-m) not matter the viewport size.

I was hesitant to change either of these but let me know if I'm misunderstanding.

Reply:
Updated this to use static units.
I made it so it just has a separate font-size declaration if >900px.
This seems okay on mobile view and desktop/tablet.